### PR TITLE
Add content on Notebooks under `Edit the code`

### PR DESCRIPTION
**Pull request by praneesha**
Add content on Notebooks from the [VS Code marketplace docs](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) under `Edit the code`

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -63,6 +63,8 @@ nav:
       - Code navigation: edit-the-code/code-navigation.md
       - Code actions: edit-the-code/code-actions.md
       - Diagnostics: edit-the-code/diagnostics.md
+      - Commands: edit-the-code/commands.md
```
Comment (anupama-pathirage) [inline en/mkdocs.yml:66]: Is this a new page, seems this PR doesn't have commands.md

Comment (praneesha) [reply en/mkdocs.yml:66]: These are the content that was moved from the VS Code marketplace. The `commands.md` file is added via https://github.com/wso2/docs-ballerina-vscode-extension/pull/102.

File: en/docs/edit-the-code/notebooks.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Ballerina notebooks can be created in VS Code using the `.balnotebook` extension for filename. Markdown and Ballerina code snippets are supported by Ballerina notebook cells.
+
+!!! Info
+    Make sure your VS Code version is `1.67.0` or higher when using Ballerina notebooks.
+
+## Create a Notebook 
+
+The command below creates a notebook with the given filename.
+
+<img src="../img/edit-the-code/notebooks/notebook-create.gif" width="100%" />
```
Comment (anupama-pathirage) [inline en/docs/edit-the-code/notebooks.md:12]: Use the standard image formatting format with CSS styles. i.e. use `{.cInlineImage-half}` and `{.cInlineImage-full}` appopriately

Comment (praneesha) [reply en/docs/edit-the-code/notebooks.md]: Updated.

Comment (praneesha) [reply en/docs/edit-the-code/notebooks.md]: Done.

**Pull request merged**